package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Music Media Web App",
   "main": "index.js",
   "dependencies": {
-    "express": "^4.15.4"
+    "express": "^4.15.4",
+    "prettier": "^1.7.0"
   },
```

Comment:
@trilxs Please resolve the other Github pull request

Also, did you change the settings for Prettier to 5 white spaces when indenting? Please reply back to the Trello ticket with these instructions so I can do them too!

Reply:
Hmm, I didn't change anything. I just did "npm install prettier". I'll add a description for my process for the next ones.